pom.xml

```diff
@@ -1,29 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.52</version>
+    <version>4.79</version>
+    <relativePath />
   </parent>
 
   <artifactId>p4</artifactId>
-  <version>1.15.2-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
   <name>P4 Plugin</name>
   <description>Perforce Client plugin for the Jenkins SCM provider.</description>
-  <url>https://github.com/jenkinsci/p4-plugin</url>
+  <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
 
   <properties>
-    <jenkins.version>2.361.4</jenkins.version>
+    <revision>1.15.2</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.414.3</jenkins.version>
+    <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
```

Comment:
Latest recommendation from [the archetype](https://github.com/jenkinsci/archetypes).